lib/generator.js

```diff
@@ -50,22 +51,34 @@ const renderFile = (filePath, data) => new Promise((resolve, reject) => {
  */
 const generateFile = options => new Promise((resolve, reject) => {
   const templates_dir = options.templates_dir;
+  const template_config = options.template_config;
   const target_dir = options.target_dir;
   const file_name = options.file_name;
   const root = options.root;
   const data = options.data;
 
-  renderFile(path.resolve(root, file_name), data)
-    .then((parsed_content) => {
-      const template_path = path.relative(templates_dir, path.resolve(root, file_name));
-      const generated_path = path.resolve(target_dir, template_path);
+  if (isFileIgnored(file_name, template_config.ignore)) {
```

Comment:
Maybe we should come up with a better name here and in the config file? Ignoring sounds like it shouldn't be even copied.